apps/src/makerlab/TouchSensor.js

```diff
@@ -0,0 +1,64 @@
+/** @file Wrapper for playground-io Touchpad to expose individual pads */
+import {EventEmitter} from 'events'; // provided by webpack's node-libs-browser
+
+/**
+ * Given a playground-io Touchpad controller, monitors touch events and forwards
+ * 'down' and 'up' events for a particular pin.
+ * @constructor
+ * @param {number} pinIndex - Which pin to pay attention to and forward events for.
+ * @param {Touchpad} touchpadsController - playground-io Touchpad controller.
+ */
+export default class TouchSensor extends EventEmitter {
+  constructor(pinIndex, touchpadsController) {
+    super();
+
+    /**
+     * @private {number} Which single pin this controller will monitor and
+     *          forward events for.
```

Comment:
I habitually indent past the `@tag` for line wraps in JSDoc comments ([have been doing this for a while](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/netsim/netsim.js#L588)) because I think it reads nicely in parameter lists, but I don't know what our preferred style is.  It looks like [Use JSDoc](http://usejsdoc.org/about-block-inline-tags.html) doesn't illustrate any indenting.  Do you have a preference?